### PR TITLE
adding force refresh script

### PR DESCRIPTION
**Pull request by aaronhmiller**
## Description

This script use the option-command-R keyboard shortcut to force refresh a Safari page. It arose out of a need to get a current CI badge status when using Github Action workflows because I noticed that the badge would not reflect the current status unless cache was cleared and the page refreshed. Normally that requires enabling Developer Menu and opt-cmd-E and cmd-R, but opt-cmd-R seems to do both and doesn't require enabling Developer Mode. Having this in Raycast is just one less thing to remember and it's nice to have it stored centrally.

## Type of change

- [X] New script command

## Screenshot

Not much to see here...it just force refreshes the frontmost safari page. I added a comment about this being undocumented and therefore subject to change w/o notice. 

## Dependencies / Requirements

none

## Checklist

- [X] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Review by unnamedd — APPROVED** [review]
Looks good to me, thanks for your first contribution @aaronhmiller!

File: commands/apps/safari/safari-clear-cache-reload.applescript [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/osascript
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Clear cache and refresh page
```
Comment (fe9lix) [inline commands/apps/safari/safari-clear-cache-reload.applescript:5]: Could you rename this to "Clear Cache and Refresh Page" to be consistent with Raycast commands?

Comment (aaronhmiller) [reply commands/apps/safari/safari-clear-cache-reload.applescript:5]: Will do!

**Comment by aaronhmiller** [comment]
anything else I should do to make this mergeable?

**Comment by unnamedd** [comment]
@aaronhmiller, nope. If you have changed everything was asked, then just wait. As I'm not part of the team, I cannot merge it, but they would do it soon.

**Review by fe9lix — APPROVED** [review]

**Pull request merged**